docker/apply/forms-json/translators-interpreters.json

```diff
@@ -601,10 +639,28 @@
       ],
       "next": [
         {
-          "path": "/have-you-provided-services-to-british-nationals-before"
+          "path": "/are-you-a-sworn-translator-and-interpreter-in-your-country"
         }
       ]
     },
+    {
+      "path": "/are-you-a-sworn-translator-and-interpreter-in-your-country",
+      "title": "Are you a certified or sworn translator in your country?",
+      "components": [
+        {
+          "name": "swornTranslationsForInterpreters",
+          "options": {},
+          "type": "YesNoField",
+          "hint": "If sworn or certified translation is not used in your country, select 'No'.",
+          "title": "Are you a certified or sworn translator in your country?",
+          "schema": {}
+        }
+      ],
+      "next": [
+        { "path": " /are-you-a-sworn-interpreter-in-your-country" },
```

Comment:
Sorry, I've just looked at this properly.. should this be conditional depending on if a user has selected that they're a translator/interpreter? 
(also there is whitespace here, which should be fine but should be cleaned up anyway)